postgres/standby_setup.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+
+if [ ! -s "$PGDATA/PG_VERSION" ]; then
+
+	pg_basebackup -d "host=$MASTER_NODE password=$REPLICATION_PASSWORD port=5432 user=$REPLICATION_USER" -D ${PGDATA} -vP
+
+	set -e
+
+	cat > ${PGDATA}/recovery.conf <<-EOF
+	standby_mode = on
+	primary_conninfo = 'host=$MASTER_NODE password=$REPLICATION_PASSWORD port=5432 user=$REPLICATION_USER'
+	trigger_file = '/tmp/touch_to_promote_to_master'
```

Comment:
should document this somewhere. "if the database fails, run this command to promote the replica"